nipype/interfaces/freesurfer/petsurfer.py

```diff
@@ -0,0 +1,591 @@
+    )
+
+    keep_cc = traits.Bool(
+        argstr="--keep-cc", desc="do not relabel corpus callosum as WM"
+    )
+
+    dmax = traits.Float(
+        argstr="--dmax %f",
+        desc="distance threshold to use when subsegmenting WM (default is 5)",
+    )
+
+    ctx_annot = traits.Tuple(
+        traits.String,
+        traits.Int,
+        traits.Int,
+        argstr="--ctx-annot %s %i %i",
+        desc="annot lhbase rhbase : annotation to use for cortical segmentation (default is aparc 1000 2000)",
+    )
+
+    wm_annot = traits.Tuple(
+        traits.String,
+        traits.Int,
+        traits.Int,
+        argstr="--wm-annot %s %i %i",
+        desc="annot lhbase rhbase : annotation to use for WM segmentation (with --subsegwm, default is lobes 3200 4200)",
+    )
+
+    output_usf = traits.Int(
+        argstr="--output-usf %i",
+        desc="set output USF different than USF, mostly for debugging",
+    )
+
+    head = traits.String(
+        argstr="--head %s", desc="use headseg instead of apas+head.mgz"
+    )
+
+    subseg_cblum_wm = traits.Bool(
+        argstr="--subseg-cblum-wm", desc="subsegment cerebellum WM into core and gyri"
+    )
+
+    no_pons = traits.Bool(
+        argstr="--no-pons", desc="do not add pons segmentation when doing ---xcerseg"
+    )
+
+    no_vermis = traits.Bool(
+        argstr="--no-vermis",
+        desc="do not add vermis segmentation when doing ---xcerseg",
+    )
+
+    ctab = File(exists=True, argstr="--ctab %s", desc="colortable")
+    no_seg_stats = traits.Bool(
+        argstr="--no-seg-stats", desc="do not compute segmentation stats"
+    )
+
+
+class GTMSegOutputSpec(TraitedSpec):
+    out_file = File(exists=True, desc="GTM segmentation")
+
+
+class GTMSeg(FSCommand):
+    """create an anatomical segmentation for the geometric transfer matrix (GTM).
+
+    Examples
+    --------
+    >>> gtmseg = GTMSeg()
+    >>> gtmseg.inputs.subject_id = 'subject_id'
+    >>> gtmseg.cmdline
+    'gtmseg --o gtmseg.mgz --s subject_id'
+    """
+
+    _cmd = "gtmseg"
+    input_spec = GTMSegInputSpec
+    output_spec = GTMSegOutputSpec
+
+    def _list_outputs(self):
+        outputs = self.output_spec().get()
+        outputs['gtm_file'] = os.path.join(
+            self.inputs.subjects_dir, self.inputs.subject_id, 'mri', 'gtmseg.mgz'
+        )
+        return outputs
+
+    def _format_arg(self, name, spec, value):
+        return super(GTMSeg, self)._format_arg(name, spec, value)
+
+
+class GTMPVCInputSpec(FSTraitedSpec):
+
+    in_file = File(
+        exists=True,
+        argstr="--i %s",
+        mandatory=True,
+        copyfile=False,
+        desc="input volume - source data to pvc",
+    )
+
+    frame = traits.Int(
+        argstr="--frame %i", desc="only process 0-based frame F from inputvol"
+    )
+
+    psf = traits.Float(argstr="--psf %f", desc="scanner PSF FWHM in mm")
+
+    segmentation = File(
+        argstr="--seg %s",
+        exists=True,
+        mandatory=True,
```

Comment:
These should be xor'd to ensure --reg LTA and --regheader aren't both used.

Reply:
solved via [bd2975e](https://github.com/nipy/nipype/pull/3437/commits/bd2975e5ea1a8ef602de03ec2fc34d2c8e354a8a)